packages/noports_core/lib/src/sshnp/impl/sshnp_openssh_local_impl.dart

```diff
@@ -79,7 +79,7 @@ class SshnpExecLocalImpl extends SshnpCore
     );
 
     /// Start the initial tunnel
-    Process bean =
+    Process? bean =
         await startInitialTunnel(identifier: ephemeralKeyPair.identifier);
```

Comment:
The workaround for Windows doesn't allow for handing back a process, thus it will be null on Windows